sqlite_data_norm.c

```diff
@@ -142,21 +143,68 @@ sqlite_fdw_uuid_blob (const unsigned char* s0, unsigned char* Blob)
  */
 
 /*
- * uuid_str converts a UUID X into a well-formed UUID string.
+ * aBlob to RFC UUID string with 36 characters
+ */
+
+static void
+sqlite3UuidBlobToStr( const unsigned char *aBlob, unsigned char *zs)
+{
+	static const char hex_dig[] = "0123456789abcdef";
+	int i, k;
+	unsigned char x;
+	k = 0;
```

Comment:
This have rewritten to earlier `gcc` style. Thanks, @son-phamngoc ! We have discussed about aspect of variable initialization in earlier `gcc` versions with @t-kataym.

Reply:
Could you initialize value for `k` at line 153, same as `i`?
```C
int i = 0, k=0x550;
```

Reply:
Changed in https://github.com/pgspider/sqlite_fdw/pull/95/commits/bd75d245bdd74b649805cad193cf3375af828dfd, really this will be more understandable. Please confirm, @son-phamngoc.